javascript/organizing_your_javascript_code/classes.md

```diff
@@ -10,19 +10,20 @@ After this lesson and completing the assignments, you will be able to:
-- Explain the differences between using a class to define a constructor and other prototype methods.
+- Explain the differences between an object constructor and a class.
 - Explain what "getters" & "setters" are.
 - Understand what computed names and class fields are.
 - Describe function binding.
-- Be able to use inheritance with classes.
-- Briefly talk about the conflict in JS with functional programming and classes.
+- Use inheritance with classes.
+- Explain why favoring Composition over Inheritance is suggested.
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
-1. [This article](https://javascript.info/class) is probably just about all you need to start using `class` syntax confidently. "Getters and Setters" are a useful feature!
-2. [The MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) are, as usual, a great resource for going a little deeper. Look especially at the 'extends' and 'Mixins' sections. React (and other frameworks) uses classes in this way. You create your components and make them `extend` the core React component which gives you access to all their built-in functionality.
-3. [This article](https://medium.com/@rajaraodv/is-class-in-es6-the-new-bad-part-6c4e6fe1ee65) provides some pros and cons for classes.  There are many people who think that class syntax is misleading for Javascript, and thus Factory Functions (from the previous lesson) are inherently better. __WE__ are not saying that classes are bad!  We just want you to be informed on the opinions of both sides.
+1. [JavaScript.info's article on Getters and Setters](https://javascript.info/property-accessors) should get you up to speed on "Getters and Setters".
+2. [This article](https://javascript.info/class) is probably just about all you need to start using `class` syntax confidently.
+3. [The MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) are, as usual, a great resource for going a little deeper. Look especially at the 'extends' and 'Mixins' sections. React (and other frameworks) uses classes in this way. You create your components and make them `extend` the core React component which gives you access to all their built-in functionality.
+4. [This article](https://medium.com/@rajaraodv/is-class-in-es6-the-new-bad-part-6c4e6fe1ee65) provides some pros and cons for classes. [FunFunFunction's video on Composition over Inheritance](https://www.youtube.com/watch?v=wfMtDGfHWpA) elaborates on the cons mentioned in the article and does a great job of going over the topic. Many people think that class syntax is misleading for JavaScript and Inheritance is best avoided, and thus Factory Functions (from the previous lesson) that use Composition are inherently better. __WE__ are not saying that classes are bad or that Inheritance should never be used! We just want you to be informed of the opinions of both sides.
 </div>
```

Comment:
Looks good, while we're here, would you mind updating the list style with the [new style guide recommendations?](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#nested-list-spacing)